spec/parsePdr/ParsePdr.input.template.json

```diff
@@ -31,13 +33,13 @@
     "StatusReport": {},
     "ParsePdr": {
       "provider": "{{$.meta.provider}}",
-      "collection": "{{$.meta.collection}}",
-      "bucket": "{{$.meta.buckets.internal}}",
+      "internalBucket": "{{$.meta.buckets.internal}}",
```

Comment:
So this is changing back to bucket now?

Reply:
Marc already has a PR out for it

Reply:
I think he should merge then we should just change it back, then get this PR merged and do the release. From there we can talk about moving to the Cumulus repo, although reading your process it doesn't really seem that bad. Slightly more cumbersome than having it in the Cumulus repo.

Reply:
The `internalBucket` to `bucket` change has been made to the master branch of the cumulus repo.  The `CUMULUS-450` branch of this repo has also been updated with that change.

Reply:
changed.